### ISSUE TITLE
Examples are out of date

### ISSUE THREAD

**Issue by BopItFreak** [cschuller/gamesense-client#19]
Tried to run some of the examples, got 
`gamesense.GameEventHandler is not a constructor`

Turns out the GameEventHandler class doesn't exist. Are the examples out of date?

**Comment by BopItFreak** [comment]
@DaanWet

**Comment by DaanWet** [comment]
Yes examples are out of date, you should now use one of the following event handlers: 

- ColorEventHandler
- FullColorEventHandler
- ScreeneventHandler
- TactileEventHandler

I will try to update the examples in the near future, for now look ath [my other repo](https://github.com/DaanWet/Ets2-Steelseries-Engine) for an up to date (altough not well documented) example

**Comment by mathbles703** [comment]
@DaanWet I cannot seem to get the examples to run even with the new event handlers. I see the events in my console but my keyboard does nothing. Any suggestions?
